migration-engine/migration-engine-tests/tests/native_types/mssql.rs

```diff
@@ -1973,7 +1973,7 @@ fn risky_casts_with_existing_data_should_warn(api: TestApi) {
 
             api.schema_push_w_datasource(dm1).send().assert_green();
 
-            let insert = Insert::single_into((api.schema_name(), "A")).value("x", seed.clone());
+            let insert = Insert::single_into((api.schema_name(), "A".to_owned())).value("x", seed.clone());
```

Comment:
why?

Reply:
It's unfortunate, but since we generate the connection info on the fly (instead of getting it from the connector), we have to return the schema name owned from `TestApi::schema_name()`. Since `quaint::ast::Table<'_>` only implements `From<(&str, &str)>` and `From<(String, String)>` we have to make the table name owned as well. `From<(String, &str)` would work, but it didn't seem worth the trouble.

Reply:
There's almost certainly a better way to be found by spending some more time on this, but it didn't seem worth the time to save a couple allocations in ~ a dozen tests.

Reply:
Yeah, tests only so it's ok. It is still possible:

```rust
let table = Table::from("A").database(api.schema_name());
```

You don't need to do this, and I should maybe reconsider naming that `database` method to something that reflects better what it does.

Reply:
Oh, I didn't know about that `database` method. That would fix it.